cmd/openshift-install/migrate/azure/migrate.go

```diff
@@ -0,0 +1,53 @@
+package azure
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/pkg/migrate/azure"
+)
+
+var (
+	azureMigrateOpts struct {
+		zone              string
+		resourceGroup     string
+		virtualNetwork    string
+		vnetResourceGroup string
+		link              bool
+	}
+)
+
+func runMigrateAzurePrivateDNSMigrateCmd(cmd *cobra.Command, args []string) error {
+	if azureMigrateOpts.zone == "" {
+		logrus.Fatal("zone is a required argument")
```

Comment:
you can return an error here instead of fatal error..

Reply:
Fixed.